webapp/channels/.eslintrc.json

```diff
@@ -20,6 +20,10 @@
     "react": {
         "pragma": "React",
         "version": "detect"
+    },
+    "formatjs": {
+      "additionalFunctionNames": ["defineMessage","localizeMessage"],
+      "additionalComponentNames": ["FormattedMarkdownMessage"]
```

Comment:
@calebroseland amazing work on this one, but we can have the replacement of `FormattedMarkdownMessage` as a community campaign to remove them progressively

Reply:
Yeah, was temporary and will remove, was mainly experimenting with this—some of the linting rules don't work well or have internal breakdowns in parsing a few of the `FormattedMarkdownMessage` messages.